tests/core/commands/test_storage.py

```diff
@@ -95,3 +95,130 @@ def test_lfs_storage_unpushed_clean(runner, project, client_with_remote):
 
     assert 0 == result.exit_code
     assert "These paths were ignored as they are not pushed" in result.output
+
+
+def test_lfs_fix(runner, project, client):
+    """Test ``renku storage migrate`` command for large files in git."""
+
+    with (client.path / "dataset_file").open("w") as fp:
+        fp.write("dataset file")
```

Comment:
Just a small suggestion.

Since, `client.path` is `Path` object, you could simplify this by saying 

```
(client.path / "dataset_file").write_text("dataset file")
```

then all those file creation could be something like

```
for _file in ["dataset_file", "workflow_file", "regular_file"]:
    (client.path / _file).write_text(_file.replace("_", " "))
```

where, `.replace` could be omitted.